src/main/java/sx/blah/discord/util/LogMarkers.java

```diff
@@ -59,7 +59,12 @@
 	 * The marker for websocket keepalive actions.
 	 * It is a child of {@link #WEBSOCKET} and {@link #VOICE_WEBSOCKET}.
 	 */
-	KEEPALIVE(WEBSOCKET, VOICE_WEBSOCKET);
+	KEEPALIVE(WEBSOCKET, VOICE_WEBSOCKET),
+	/**
+	 * The marker for all websocket-related logging.
+	 * It is a child of {@link #API}.
+	 */
+	CLOUDFLARE(WEBSOCKET);
```

Comment:
This isn't necessary for just one log message, use the provided UTIL marker. The javadoc on this is also wrong